sda/host.go

```diff
@@ -612,3 +617,63 @@ func (h *Host) Rx() uint64 {
 func (h *Host) Address() string {
 	return h.workingAddress
 }
+
+//Received gives packets received
+func Received(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Rx()
+	}
+	return a
+}
+
+//Sent gives packets sent
+func Sent(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Tx()
+	}
+	return a
+
+}
+
+//Host is the host
+func id(n map[network.ServerIdentityID]network.SecureConn) string {
+	var a string
+	for _, value := range n {
+		a = value.Local()
+	}
+	return a
+
+}
+
+//Remote is who the host is connected to
+func Remote(n map[network.ServerIdentityID]network.SecureConn) string {
+	var a string
+	for _, value := range n {
+		a = a + value.Remote() + "\n"
+	}
+	return a
+}
+
+//GetStatus is a function
+func (h *Host) GetStatus() Status {
+	m := make(map[string]string)
+	m["Host"] = id(h.Connections)
+	m["Connections"] = Remote(h.Connections)
+	s := fmt.Sprintf("%d", len(h.Connections))
+	m["Total"] = s
+	t := fmt.Sprintf("%d", Received(h.Connections))
+	m["Packets Received"] = t
+	x := fmt.Sprintf("%d ", Sent(h.Connections))
```

Comment:
Instead of having all these functions that loop over the connections, I prefer you write this like
```
nbr := len(h.Connections)
remote := make([]string, nbr)
rx := 0
for i, c := range h.Connections{
    remote[i] = c.Remote
    rx += c.Rx()
}
m["Connections"] = strings.Join(remote, "\n")
```
and add in the for-loop all other things you need. This makes the code clearer and removes the 'global' methods.